README.md

```diff
@@ -1 +1,38 @@
-# simple-calculator
\ No newline at end of file
+# Calculator ++
+***
+## OVERVIEW
+A fully complete basic calculator created in the C/C++ programming language that is both entertaining and simple to construct. It resembles an elementary/middle school calculator which allows us to Add, Subtract, Multiply, Divide, Square and get the Square Root of a number.
+***
+## REASON FOR MAKING
+This project was created for my Morgan State University peers who are just starting to learn how to code. Making a calculator, in my opinion, is the ideal first project in any language since you can learn a ton of concepts like:
+
+* **Object-Oriented Programming**
+* **Handling User Input and Output**
+* **Implementing Functions**
+* **Control Structures (loops/conditional statements)**
+* **Validating User Input**
+* **Testing and Debugging**
+* **Comfortability with Git**
+  * **_git commit_**
+  * **_git push/pull_**
+  * **_git branches_**
+  * **_git pull requests_**
+  * **_advanced git commands such as rebase/cherry-picking_**
```

Comment:
Im not sure you can put down testing and debugging when you have no unit-tests it gives the wrong impression. Also feel like you are overselling the git part its good to learn these things but a potential employer seeing this code will be able to see that you know git by the looking at the commit history and seeing that master is all merge commits.